lib/ansible/plugins/__init__.py

```diff
@@ -70,9 +73,11 @@ def matches_name(self, possible_names):
         return bool(possible_fqcns.intersection(set(self.ansible_aliases)))
 
     def get_option(self, option, hostvars=None):
+
+        templar = self.get_templar()
```

Comment:
How about moving this into the `if`, so that the two `getattrs()` only need to be evaluated when `option not in self._options`?

Reply:
done